gitops/azure-devops/build.sh

```diff
@@ -97,13 +118,12 @@ function fab_generate() {
     fi
 
     echo "FAB GENERATE COMPLETED"
-
     set +e
 
     # If generated folder is empty, quit
     # In the case that all components are removed from the source hld,
     # generated folder should still not be empty
-    if find "$HOME/generated" -mindepth 1 -print -quit 2>/dev/null | grep -q .; then
+    if find "generated" -mindepth 1 -print -quit 2>/dev/null | grep -q .; then
```

Comment:
I'm assuming we are already in the directory for `generated` and don't need $HOME

Reply:
Yes, we don't need it, also backwards compatible (verified)